tns-core-modules/application/application.android.ts

```diff
@@ -177,11 +177,13 @@ export function run(entry?: NavigationEntry | string) {
     _start(entry);
 }
 
-export function addCss(cssText: string): void {
+export function addCss(cssText: string, scoped?: boolean): void {
```

Comment:
```suggestion
export function addCss(cssText: string, attributeScoped?: boolean): void {
```

This implementation is needed only when the css is global, but scoped by attributes (like in Angular and Vue). To differentiate this from the {N} Core implementation with style scope objects I think it's a better naming.